dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/bnd.bnd

```diff
@@ -102,11 +109,11 @@ DynamicImport-Package: \
 Private-Package: io.openliberty.persistence.internal.helper
 
 Include-Resource: \
-  @${repo;org.eclipse.persistence.asm;3.0.0}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.corba;3.0.0}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.core;3.0.0}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.jpa;3.0.0}!/!META-INF/maven/*,\
-  @${repo;org.eclipse.persistence.jpa.jpql;3.0.0}!/!META-INF/maven/*
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.asm;${eclFullVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.corba;${eclFullVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.core;${eclFullVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa;${eclFullVersion}}!/!META-INF/maven/*,\
+  @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;${eclFullVersion}}!/!META-INF/maven/*
```

Comment:
Ideally these Include-Resources would have the EXACT and would have the right full version.  Looks like it could be 3.0.0.M1 or 3.0.0.SNAPSHOT depending on which version of 3.0.0 is being used.

Reply:
Yeah, I'm having some troubles getting this to work, might be because of how I built and generated the eclipselink 3.0 bundle.  I had overridden the file names maven generated to use a version of 3.0.0-R1.5, however I didn't realize until too late that the manifest file still listed a version of 3.0.0.SNAPSHOT.  I'm not sure that a bnd.bnd file change can work around this, might need to rebuild and reupload the eclipselink binary.